app/info.md

```diff
@@ -0,0 +1,56 @@
+### Model Description
+
+## Summary
+
+The principle of the estimation is:
+
+1. Augment the number of admissions using the reporting. This is currently done by
+
+ n<sub>aug</sub> = n<sub>reported</sub> / % reported
+ 
+2. Use a log-linear model, parametrised through the doubling time, to simulate epidemic trajectories; see for instance Jombart et al 2020; this is implemented by the RECON package projections.
+
+3. For each admission, simulate duration of hospitalisation from provided distribution.
+
+4. Count beds for each day and simulation.
+
+
+### Model Parameters
+
+This section contain information on the various parameters. We use these
+data to generate a distribution, with discretisation when needed.
+
+* **Duration of hospitalisation**:
+    + non-critical care: discretised Weibull(shape:, scale:) to aim for a median of 11
```

Comment:
I'm unclear about why these are called "non-critical care" and "critical care" distributions: don't we just use them as "short" or "long" stay in the figures? We are predicting total duration hospitalisation or time in ICU / critical care beds?

Reply:
They come from a paper which estimated the LoS for critical care / normal hospitalisation.